### PR TITLE
new version and add author to META.json

### PR DESCRIPTION
**Pull request by PerikAdjust**
I tried to publish this first extension on PGXN and I got the following error: "User "adjustgmbh" does not own all provided extensions". After some research I have the impression that the PGXN account and the META.json is a link at the author level, which is why I chose this author name.

### CODE REVIEW

**Comment by einhverfr** [comment]
Sorry, but that's not going to be enough.  we need to talk t oDave Wheeler. I put this on pgxn when there wasn't a lot of process around this.

If if you want I can push new versions and you can be upstream.  your choice.

**Comment by einhverfr** [comment]
Please see my list of extension published at https://www.pgxn.org/user/einhverfr  We should probably figure out how to hand these over.

I can reach out to ads to make this happen if you like.

**Comment by einhverfr** [comment]
But maybe if you ant to do this we should bump it to 0.0.3 (or even 1.0.0)?  

It's worth noting that with new versions of Postgres, this extension is a lot less useful than it used to be though.

**Comment by PerikAdjust** [comment]
First of all, I'm sorry I didn't find Adjust's public extensions before. When you search for "ajbool" in the pgxn search bar it doesn't come up. 
I now understand why most of our public extensions already had a working META.json.

**Comment by PerikAdjust** [comment]
To be honest I don't know if the problem comes from the other provider but I had this error when trying to publish ajbool. 
image

According to my research the verification of the non-corelation of the META.json and the PGXN account of adjust could have been the consequence.

File: META.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
    "name": "ajbool",
    "abstract": "Three valued boolean with pseudo-NULL for primary keys",
+   "author": "adjustgmbh",
    "version": "0.0.2",
    "maintainer": [ "Chris Travers <chris.travers@adjust.com>", "Manuel Kniep <rapimo@adjust.com>" ],
```
Comment (einhverfr) [inline META.json:6]: You want to change this too.  I don't imagine Manuel is much of a maintainer anymore and I certainly am not,

**Review by einhverfr — CHANGES_REQUESTED** [review]

**Comment by einhverfr** [comment]
Your actual problem was that I published these back on pgxn in 2017 and 2018 when Adjust had no process for setting up some sort of company account.

The ownership system is entirely done manually by Dave Wheeler.  So we should produce a list of extensions to transfer, a proper account to transfer them to, and get in touch with him about it.

**Comment by andreasscherbaum** [comment]
ok, how about I take this list, we review it, and then I send a mail to David with the list and copy all parties involved for approval?

**Comment by einhverfr** [comment]
Sounds good.

File: META.json (hunk 1 above)
Comment (PerikAdjust) [reply META.json:6]: Yes, especially after talking to Arthur, I don't think there's no maintener specially dedicated to it now.

**Review by andreasscherbaum — APPROVED** [review]

**Pull request merged**